stories/3-Primitives.stories.js

```diff
@@ -1,58 +1,39 @@
 import React from "react";
 import Background from "../src/styles/Background";
-import Gradient from "../src/styles/Gradient";
-import Card from "../src/components/Card";
 import Avatar from "../src/styles/Avatar";
 import {face} from "../src/images";
+import darkMode from "../src/images/darkMode.svg";
+import lightMode from "../src/images/lightMode.svg";
+import themeAuto from "../src/images/themeAuto.svg";
 
 export default {
   title: "Primitives",
 };
 
-export const GradientBackground = () => (
-  <React.Fragment>
-    <Gradient>
-      <div style={{height: 1024, width: 498}} />
-    </Gradient>
-  </React.Fragment>
-);
-
 export const ProfileAvatar = () => (
   <Background style={{height: 1024, padding: "10px"}}>
     <Avatar src={face} alt="profile pic" />
```

Comment:
See this as a rectangle rather than a square. Are you seeing this?

<img width="535" alt="Screen Shot 2021-06-16 at 9 39 25 PM" src="https://user-images.githubusercontent.com/5713670/122332785-e46e4380-ceeb-11eb-8952-2e31921beef3.png">

Reply:
> See this as a rectangle rather than a square. Are you seeing this?
> 
> <img alt="Screen Shot 2021-06-16 at 9 39 25 PM" width="535" src="https://user-images.githubusercontent.com/5713670/122332785-e46e4380-ceeb-11eb-8952-2e31921beef3.png">

Updated this so it is a square instead of a rectangle and will be added upon completion of all the other suggestions.